package.json

```diff
@@ -30,32 +52,39 @@
     "@metamask/eslint-config-jest": "^7.0.0",
     "@metamask/eslint-config-nodejs": "^7.0.1",
     "@metamask/eslint-config-typescript": "^7.0.1",
-    "@types/jest": "^26.0.13",
-    "@typescript-eslint/eslint-plugin": "^4.21.0",
-    "@typescript-eslint/parser": "^4.21.0",
-    "eslint": "^7.23.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^24.3.4",
+    "@open-rpc/typings": "^1.9.0",
+    "@types/jest": "^24.0.13",
+    "@types/pump": "^1.1.1",
+    "@types/react-dom": "^16.8.4",
+    "@typescript-eslint/eslint-plugin": "^4.28.3",
+    "@typescript-eslint/parser": "^4.28.3",
+    "eslint": "^7.30.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jest": "^24.3.6",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "jest": "^26.4.2",
+    "eslint-plugin-prettier": "^3.4.0",
+    "html-webpack-plugin": "^5.3.2",
+    "jest": "^24.9.0",
+    "node-polyfill-webpack-plugin": "^1.1.4",
     "prettier": "^2.2.1",
     "prettier-plugin-packagejson": "^2.2.11",
+    "react-scripts": "^3.4.4",
     "rimraf": "^3.0.2",
-    "ts-jest": "^26.3.0",
-    "typescript": "^4.2.4"
-  },
-  "engines": {
-    "node": ">=12.0.0"
-  },
-  "publishConfig": {
-    "access": "public",
-    "registry": "https://registry.npmjs.org/"
+    "ts-jest": "^24.0.2",
+    "ts-loader": "^9.2.3",
+    "tslint": "^5.17.0",
+    "typescript": "^3.9.7",
```

Comment:
This seems pretty old :thinking: I don't think this is compatible with our TypeScript lint config for one thing.